flags.go

```diff
@@ -492,8 +495,8 @@ func populateFlags(c *cli.Context) (flags *flagStorage, err error) {
 		// Tuning,
 		MaxRetrySleep:              c.Duration("max-retry-sleep"),
 		StatCacheCapacity:          c.Int("stat-cache-capacity"),
-		StatCacheTTL:               c.Duration("stat-cache-ttl"),
-		TypeCacheTTL:               c.Duration("type-cache-ttl"),
+		StatCacheTTL:               minTTL,
```

Comment:
also you can add logic in validateFlags function to ensure both stat-cache-ttl and type-cache-ttl have same values.

Reply:
Done.